packages/gatsby/src/schema/data-tree-utils.js

```diff
@@ -244,6 +244,9 @@ const extractFieldExamples = (
   const example = {}
   for (let key of allKeys) {
     if (ignoreFields.includes(key)) continue
+    // Unfortunately an empty string as an Object key is perfectly valid JavaScript
+    // We ignore these because our schema generation requires keys to have names
```

Comment:
it's requirement of `graphql` (not ours :) )

Reply:
Updated the comment